pallets/amm/src/lib.rs

```diff
@@ -91,6 +91,11 @@ pub mod pallet {
         #[pallet::constant]
         type ProtocolFee: Get<Perbill>;
 
+        /// Minimum amount of liquidty needed to init a new pool
+        /// this amount is burned when the pool is created
```

Comment:
can you explain also why do we need `MinimumLiquidity`, why this can prevent from attacks in uniswap